plugins/prometheus.py

```diff
@@ -35,6 +43,101 @@ def __init__(self, *args, **kwargs):
         self.consumer.daemon = True
         self.consumer.start()
 
+
+    def cmd_show(self, event, action):
+        """Show information on projects or projects being tracked.
+        Show which projects are being tracked. 'prometheus show tracking'
+        Show which proejcts are recognised so they could be tracked. 'prometheus show projects'
+        """
+        if action in self.TRACKING:
+            return self._get_tracking(event["room_id"])
+        elif action == "projects":
+            projects = self.store.get("known_projects")
+            return "Available projects: %s" % json.dumps(projects)
+        else:
+            return "Invalid arg '%s'.\n %s" % (action, self.cmd_show.__doc__)
+
+    @admin_only
+    def cmd_track(self, event, *args):
+        """Track projects. 'prometheus track Foo "bar with spaces"'"""
+        if len(args) == 0:
+            return self._get_tracking(event["room_id"])
+
+        for project in args:
+            if not project in self.store.get("known_projects"):
+                return "Unknown project name: %s." % project
+
+        self._send_track_event(event["room_id"], args)
+
+        return "Prometheus notifications for projects %s will be displayed when they fail." % (args)
+
+    @admin_only
+    def cmd_add(self, event, project):
+        """Add a project for tracking. 'prometheus add projectName'"""
+        if project not in self.store.get("known_projects"):
+            return "Unknown project name: %s." % project
+
+        try:
+            room_projects = self.rooms.get_content(
+                event["room_id"],
+                PrometheusPlugin.TYPE_TRACK)["projects"]
```

Comment:
I agree. This was a copy and paste from the jenkins plugin with only indentation changes. I thought it was difficult to see the closing `)` too.